pkg/kvstore/etcd.go

```diff
@@ -1648,18 +1648,20 @@ func (e *etcdClient) ListPrefix(ctx context.Context, prefix string) (v KeyValueP
 // Close closes the etcd session
 func (e *etcdClient) Close() {
 	close(e.stopStatusChecker)
-	<-e.firstSession
+	sessionErr := e.waitForInitialSession(context.Background())
```

Comment:
Why is `context.Background()` OK here?

Reply:
That's right, it is not different than just blocking on a channel but we need the value of `e.sessionErr()`. The `Close()` does not have a context being passed in.